openassessment/xblock/static/css/openassessment-ltr.css

```diff
@@ -1,3 +1,3 @@
.copy,.openassessment .openassessment__steps__step .step__status .copy,.openassessment .step--response .response__submission__content .tip,.openassessment .submission__peer-evaluations__questions .question__score,.openassessment .submission__peer-evaluations__questions .answer__source .answer__source__value-with-points,.openassessment .submission__peer-evaluations__questions .answer__score,.openassessment .submission__peer-evaluations__questions .answer__score__value,.openassessment .staff-info__status__table th,.openassessment .staff-info__classifierset__table th,.openassessment .ui-hint:after,#openassessment-editor .openassessment_container_guide{font-size:12px;line-height:18px}.wrapper--xblock .message .message__title,.openassessment .staff-assessment__assessment .assessment__rubric__question .question__title__copy,.openassessment .step--peer-assessment .peer-assessment__assessment .assessment__rubric__question .question__title__copy,.openassessment .step--self-assessment .self-assessment__assessment .assessment__rubric__question .question__title__copy,.step--student-training .student-training__assessment .assessment__rubric__question .question__title__copy,.openassessment .ui-staff__title,.openassessment .ui-staff__subtitle,.openassessment .staff-assessment__display__header .staff-assessment__display__title,.openassessment .submission__answer__part__text__title,.openassessment .step--peer-assessment .peer-assessment__display__header .peer-assessment__display__title,.openassessment .step--self-assessment .self-assessment__display__title,.openassessment .submission__answer__display__title,.openassessment .submission__peer-evaluations__title,.openassessment .submission__peer-evaluations__questions .question__title__copy,.openassessment .submission__feedback__title,.openassessment .submission__feedback__title__copy,.step--student-training .student-training__display__title,.openassessment .openassessment__steps__step .step__title .step__deadline,.staff-info__student .title--sub,.wrapper--xblock .field label,.wrapper--xblock .field .label,.wrapper--xblock .field .field-group__label,.wrapper--xblock .field--radio .option__label,.wrapper--xblock .field--checkbox .option__label,.wrapper--xblock .message .message__content,.openassessment .staff-assessment__assessment .question__answers .wrapper--input .answer__label,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .wrapper--input .answer__label,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--input .answer__label,.step--student-training .student-training__assessment .question__answers .wrapper--input .answer__label,.openassessment .staff-assessment__assessment .question__answers .answer__tip,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .answer__tip,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .answer__tip,.step--student-training .stude
```

Comment:
does this file need to be committed? or does it get auto-generated when building? did any scss files change?
Same comment applies for the other .css and .min.js files in this PR

Reply:
`.min.js` need to be committed (because they are used when DEBUG == `False` in the server settings). `.css` seems that was committed wrong